### ISSUE TITLE
Add alt attribute to all images in the CMS admin

### ISSUE THREAD

**Issue by thibaudcolas** [wagtail/wagtail#5291]
Screen reader users cannot view images. Images need to either have a textual description, or an empty alt attribute (`alt=""`) for screen readers to ignore them – otherwise the image’s source URL gets read out loud.

Here are 4 occurences I know about, there might be more:

```
<img src="//www.gravatar.com/avatar/e64c7d89f26bd1972efa854d13d7dd61?s=100&amp;d=mm">, at #account-settings > span > img

Solution: add empty alt=””

<img>, at #id_image-chooser > div:nth-child(1) > div > img 

Solution: add empty alt=””

<img src="/media/images/vik.max-800x600_tWRrsIy.jpg" style="[…]"> at #wagtail > div:nth-child(19) > div:nth-child(2) > div > form > div:nth-child(2) > div:nth-child(2) > div:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > img

Solution: add empty alt=””
```

**Comment by zerolab** [comment]
I found this very insightful article on alt tags from a web developer with vision impairment - https://axesslab.com/alt-texts/ - and thought I'd leave it on this issue as a reference.

**Comment by rinti** [comment]
@zerolab That is a really good article, thanks!
